Runtime/Native/Windows/NativeClient.cs

```diff
@@ -0,0 +1,438 @@
+            const string pluginDir = "Plugins";
+            string architecture = IntPtr.Size == 8 ? "x86_64" : "x86";
+
+            string pluginPath = Path.Combine(pluginDir, architecture);
+            string pluginHandlerPath = Path.Combine(pluginPath, crashpadHandlerName);
+
+            // generate full path to .dll file in plugins dir.
+            return Path.Combine(Application.dataPath, pluginHandlerPath);
+
+        }
+        /// <summary>
+        /// Clean scoped attributes
+        /// </summary>
+        internal static void CleanScopedAttributes()
+        {
+            // cleaning scoped attributes should be skipped when 
+            // Configuration.SendUnhandledGameCrashesOnGameStartup  is set to false
+            // the reason behind this decision is to make sure user change in the configuration
+            // won't left and useless data
+            var attributesJson = PlayerPrefs.GetString(ScopedAttributeListKey);
+            if (!HasScopedAttributesEmpty(attributesJson))
+            {
+                return;
+            }
+            var attributes = JsonUtility.FromJson<ScopedAttributesContainer>(attributesJson);
+            foreach (var attributeKey in attributes.Keys)
+            {
+                PlayerPrefs.DeleteKey(string.Format(ScopedAttributesPattern, attributeKey));
+            }
+            PlayerPrefs.DeleteKey(ScopedAttributeListKey);
+        }
+
+        internal static IDictionary<string, string> GetScopedAttributes()
+        {
+            var attributesJson = PlayerPrefs.GetString(ScopedAttributeListKey);
+            if (!HasScopedAttributesEmpty(attributesJson))
+            {
+                return new Dictionary<string, string>();
+            }
+            var result = new Dictionary<string, string>();
+            var attributes = JsonUtility.FromJson<ScopedAttributesContainer>(attributesJson);
+            foreach (var attributeKey in attributes.Keys)
+            {
+                var value = PlayerPrefs.GetString(string.Format(ScopedAttributesPattern, attributeKey), string.Empty);
+                result[attributeKey] = value;
+            }
+
+            // extend scoped attributes with legacy attributes stored by Backtrace-Unity library in previous versions
+            IDictionary<string, string> legacyAttributes = new Dictionary<string, string>()
+                    {
+                        { MachineUuidKey, "guid" },
+                        { VersionKey, "application.version" },
+                        { SessionKey, BacktraceMetrics.ApplicationSessionKey }
+                    };
+
+            foreach (var legacyAttribute in legacyAttributes)
+            {
+                string legacyAttributeValue = PlayerPrefs.GetString(legacyAttribute.Key, string.Empty);
+                if (!string.IsNullOrEmpty(legacyAttributeValue))
+                {
+                    PlayerPrefs.DeleteKey(legacyAttribute.Key);
```

Comment:
Why should we delete legacy attributes from `PlayerPrefs`?

Reply:
Because this allows us to migrate legacy attributes from version 3.5.0 to new attributes system in 3.6.0 and use always standard flow. We don't need them anymore in playerprefs.